terraform/a2ha-terraform/modules/automate/files/config.toml.erb

```diff
@@ -3399,3 +3399,9 @@ rM2p0kk=
   base_path = "automate"  
 <% end %>
 
+<% if services[:postgresql][:setup_managed_services] == false && overrides[:setup_self_managed_services] == false -%>
```

Comment:
have we considered, that customer can have PG as external and OpenSearch as Chef Managed DB or vis-versa ?